terraform/deployments/opensearch/main.tf

```diff
@@ -162,10 +162,12 @@ resource "aws_opensearch_domain" "opensearch" {
 CONFIG
 }
 
-resource "aws_ssm_parameter" "opensearch_master_user" {
-  name        = "/service/${var.service}/MASTER_USER"
-  description = "opensearch_password for ${var.service} domain"
-  type        = "SecureString"
-  value       = "${local.master_user},${random_password.password.result}"
```

Comment:
Please consider using JSON for the two fields here, since secretsmanager is designed to be used that way. (Try it in the UI first and you'll see what I mean.)

Reply:
This part is deleted, unless i'm missing something

Reply:
Oh whoops, I must have fumbled the GitHub UI there 😅 Was meant to be on what's now line 171 — still valid tho I think?

Reply:
You probably want something like `secret_string = jsonencode({password = random_password.password.result})`

Reply:
oh yup i see what you mean, good point, i'll take a look

Reply:
Otherwise you'll find some clients will throw a wobbly trying to read the secretversion. (Might get away with a plain value, but it's way easier to extend and change later if you use JSON, plus you avoid any pontential compatibility issues.)

Reply:
awesome thanks for the feedback, i added the username as well since it's available (hope it's ok)

Reply:
thanks! 🙇